chia/wallet/wallet_node.py

```diff
@@ -1174,6 +1164,7 @@ async def sync_from_untrusted_close_to_peak(self, new_peak_hb: HeaderBlock, peer
             else:
                 backtrack_fork_height = new_peak_hb.height - 1
 
+            cache = self.get_cache_for_peer(peer)
```

Comment:
what does this return? it can't be a `Dict[uint32, Set[CoinState]]`, because you call `get_race_cache()` on it below

Reply:
The `PeerRequestCache` where i added `get_race_cache` in this patch here
https://github.com/Chia-Network/chia-blockchain/blob/208485075c335cfe8a715fad10192b68a0afca25/chia/wallet/wallet_node.py#L197-L200